iberz/.htaccess

```diff
@@ -1,6 +1,28 @@
-AddType text/turtle .ttl
 
+# Do not remove this line, otherwise mod_rewrite rules will stop working
+RewriteBase /
```

Comment:
Why would the rules stop working?  Rewrite base is used to reference local files and only remote redirects are used here.

Reply:
just copied the code, I can delete the comment

Reply:
Ok.  Added a suggestion. Feel free to test it out and make sure it does what you want.

Reply:
looks good!